### ISSUE TITLE
Build release electron app with auto builder

### ISSUE THREAD

**Issue by dbkr** [vector-im/riot-web#12707]
When the auto-builder finishes building a riot-web tarball (ie. making the github release) build the release electron app and upload it to packages in the same way as we do for nightly builds.

Blocked on https://github.com/neon-bindings/neon/pull/491 for 32 bit support
Split out from https://github.com/vector-im/riot-web/issues/12452

**Comment by jryans** [comment]
The Windows 32-bit portion has been deferred to after release, tracked in https://github.com/vector-im/riot-web/issues/13175.

I'm taking a slightly different approach to this for the moment to save time: the auto-builder pulls the tarball from GitHub which was previously assembled by the build laptop.

**Comment by jryans** [comment]
Fixed by https://github.com/vector-im/riot-builder/pull/23
